web/public/js/directives/cd-picture-grid/template.dust

```diff
@@ -0,0 +1,6 @@
+<div class="cd-picture-grid">
+    <a class="cd-picture-grid__item" ng-repeat="item in $ctrl.items" href="{{ item.href }}">
+        <div class="cd-picture-grid__picture" style="background-image: url({{ item.picture }});"></div>
```

Comment:
ng-style is for applying styles from objects in the scope. The items array would then have to be:
```
[{
  href: '/whatever',
  caption: 'Whatever',
  picture: {
    'background-image': 'url(/whatever.png)'
  }
}]
```

Seems easier for users of the directive to leave it as is.